docs/changes/4.4.5.md

```diff
@@ -40,6 +40,7 @@ Release date: `2020-xx-xx`
 - [NXDRIVE-2268](https://jira.nuxeo.com/browse/NXDRIVE-2268): Best efforts to center a window on the screen
 - [NXDRIVE-2276](https://jira.nuxeo.com/browse/NXDRIVE-2276): Introduce the notion of sessions
 - [NXDRIVE-2300](https://jira.nuxeo.com/browse/NXDRIVE-2300): Update the sync counter in real-time
+- [NXDRIVE-2301](https://jira.nuxeo.com/browse/NXDRIVE-2301): Handle a transfer cancel during sessions
```

Comment:
Not needed. It should have been part of NXDRIVE-2276. As it was not released, you can skip the news.